layouts/community/ortho_4x12/xyverz/config.h

```diff
@@ -0,0 +1,30 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#define MIDI_BASIC
+
+#define USE_SERIAL
+
+#define EE_HANDS
+
+#undef RGBLED_NUM
+#ifdef KEYBOARD_planck_rev5
+  #define RGBLED_NUM 10
+  #define RGB_DI_PIN D1
+#endif
+#ifdef KEYBOARD_levinson_rev2
+  #define RGBLED_NUM
```

Comment:
Hi,

Apologies for a drive-by review comment from an unknown community member... I'm learning qmk, and just happened to stumble across this while playing around some things.  Anyway....

Shouldn't RGBLED_NUM be defined to an integer value here?  Same with the lets_split variant on line 19.
I don't know what the correct RGBLED_NUM value should be for those boards, but I'm pretty sure they should have a value (like the planck_rev5 version above).

Reply:
Yeah, it should be. I missed that.

Reply:
Well crap! I guess I missed that. Sorry. I'll get that in here now.

Reply:
I updated the file in my repo. Should I make another PR, or do you want to update it on your end?